doc/api/async_context.md

```diff
@@ -224,13 +224,14 @@ to `asyncLocalStorage.getStore()` will return `undefined` until
 When calling `asyncLocalStorage.disable()`, all current contexts linked to the
 instance will be exited.
 
-Calling `asyncLocalStorage.disable()` is required before the
-`asyncLocalStorage` can be garbage collected. This does not apply to stores
-provided by the `asyncLocalStorage`, as those objects are garbage collected
-along with the corresponding async resources.
+There is no need to call this method in order to get an `asyncLocalStorage`
+instance from being garbage-collected.
```

Comment:
The wording here is a bit off. If the intention is to say that `disable()` is not needed to allow the ALS instance to be garbage collected then I would actually suggest just dropping this first sentence.

Reply:
This only works with `--async-context-frame`. Given that `--async-context-frame` is the default option now, it is simply discouraged to use `disable`.